rousseau_vote/lib/src/widgets/poll_card.dart

```diff
@@ -10,18 +10,17 @@ import 'link_flat_button.dart';
 class PollCard extends StatelessWidget {
   const PollCard(this._poll);
 
-  static const Map<PollStatus, Color> COLOR_MAPPING = <PollStatus, Color>{
-    PollStatus.PUBLISHED: PUBLISHED_ORANGE,
-    PollStatus.OPEN: OPEN_GREEN,
-    PollStatus.CLOSED: CLOSED_RED
+  static const Map<String, Color> COLOR_MAPPING = <String, Color>{
```

Comment:
So, there is a discussion we need to open here. I'm using a different logic than the web one. Instead of checking the status, I'm checking the time to determine the status. This is because, on web the voting operation is pretty live and the status is usually in sync. Here it might not be the case. That's why I was thinking to rely on the vote_starting_time etc to determine the status. This gives us some advantages.